src/test/kotlin/no/nav/familie/ef/sak/cucumber/steps/BeregningBarnetilsynStepDefinitions.kt

```diff
@@ -83,5 +84,56 @@ class BeregningBarnetilsynStepDefinitions {
 
     }
 
+    @Så("forventer vi følgende perioder med riktig grunnlagsdata")
+    fun `forventer vi følgende perioder med riktig grunnlagsdata`(dataTable: DataTable) {
+        val forventet = hentUtForventet(dataTable)
+        sjekkAtAlleFelterErSomForventet(forventet)
+        assertThat(beregnYtelseBarnetilsynResultat).size().isEqualTo(forventet.size)
+    }
+
+    private fun sjekkAtAlleFelterErSomForventet(forventet: List<ForventetPeriodeMedGrunnlag>) {
+        val sortedResultat = beregnYtelseBarnetilsynResultat.sortedBy { it.periode.fradato }
+        val sortetForventet = forventet.sortedBy { it.fraÅrMåned }
+
+        sortedResultat.forEachIndexed { idx, it ->
+            assertAllefelterErSomForventet(it, sortetForventet, idx)
+        }
+    }
+
+    private fun hentUtForventet(dataTable: DataTable) = dataTable.asMaps().map {
+        val beløp = it["Beløp"]!!.toInt()
+        val fraÅrMåned = parseValgfriÅrMåned("Fra måned", it)!!
```

Comment:
Det er lagt opp til at denne type strenger legges inn i enums. I dette tilfelle kan det opprettes en BarnetilsynDomeneParser, se VedtakDomeneParser for eksempel